lib/pp.rb

```diff
@@ -276,15 +276,20 @@ def comma_breakable
     def seplist(list, sep=nil, iter_method=:each) # :yield: element
       sep ||= lambda { comma_breakable }
       first = true
+      kwsplat = EMPTY_HASH
       list.__send__(iter_method) {|*v|
         if first
           first = false
         else
           sep.call
         end
-        RUBY_VERSION >= "3.0" ? yield(*v, **{}) : yield(*v)
+        kwsplat ? yield(*v, **kwsplat) : yield(*v)
       }
     end
+    EMPTY_HASH = if RUBY_VERSION >= "3.0"
```

Comment:
How about `EMPTY_KWHASH` ?
`EMPTY_HASH` is not an empty hash in ruby < 3.0. This hash can't be used for a purpose other than empty keyword splat.